### DISCUSSION TITLE
Jetbrains Gateway + Codespaces: Please update to latest editor versions

### DISCUSSION THREAD

**Discussion by erinmez** [community/community#66031] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi there,

I have just come back from a support request with Jetbrains. Apparently, Github needs to handle this. I would like to have the latest IntelliJ Idea **2023.2.1**  selectable in the dropdown box:

<img width="888" alt="Screenshot 2023-09-05 at 14 25 42" src="https://github.com/community/community/assets/16493423/b2983f03-64bd-4969-a2ca-b10d62f0a6ac">

**Comment by mazzarito** [comment]
Please update this...
